src/usagemetrics/user_metrics_client.py

```diff
@@ -0,0 +1,79 @@
+import json
+import sys
+from http import client
+
+import pandas
+import pandas as pd
+
+
+class UserMetricsClient:
+
+    def __init__(self, url, auth_key, project_id):
+        self.url = url
+        self.auth_key = auth_key
+        self.project_id = project_id
+
+    # Queries the user service and returns a dataframe with a column index on object type and a row index on metric
```

Comment:
there are a couple of other little dataframes we're supposed to output.  it seems maybe they are missing?